src/main/java/com/google/logsummarizer/core/BunnyFormat.txt

```diff
@@ -0,0 +1,3 @@
+=[**]= START <id> <name>
```

Comment:
no, this is documentation. From your other comment it sounds like you want a lot more documentation, though, so this would grow rather than shrink.